erpnext/selling/doctype/sales_order/sales_order.js

```diff
@@ -36,6 +36,7 @@ frappe.ui.form.on("Sales Order", {
 		})
 	},
 	customer: function (frm) {
+		frm.set_value("delivery_date", "")
```

Comment:
This should also happen on `delivery_date` if the customer is not selected

Reply:
If i do frm.set_value("delivery_date", "") on delivery_date(), then this triggers delivery_date() so it displays multiple times